src/test/java/org/opentripplanner/middleware/testutils/PersistenceTestUtils.java

```diff
@@ -267,4 +267,15 @@ static Itinerary createItinerary() {
         itinerary.legs = legs;
         return itinerary;
     }
+
+    public static void deleteOtpUser(boolean isEndToEnd, OtpUser... optUsers) {
```

Comment:
Typo, should be `otpUsers`.

Reply:
Fixed in: https://github.com/ibi-group/otp-middleware/pull/256/commits/0db59a3b2b555483e230107eb12153f15c5ac94b